website/content/docs/agent/config/config-files.mdx

```diff
@@ -1818,8 +1818,8 @@ subsystem that provides Consul's service mesh capabilities.
     geo location or datacenter, dc:sfo). By default, this is left blank and not used.
 
   - `disable_hostname` ((#telemetry-disable_hostname))
-    This controls whether or not to prepend runtime telemetry with the machine's
-    hostname, defaults to false.
+    This controls whether or not to prepend runtime telemetry of gauge type with the
+    machine's hostname, defaults to false (prepend hostname).
```

Comment:
```suggestion
    Set to `true` to stop prepending the machine's hostname to gauge-type metrics. Default is `false`.
```
Sort of a backward UX. Using direct language and active voice should make this at least a bit clearer.